ex10/index.js

```diff
@@ -0,0 +1,11 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert a integer number:");
+const num = Number(prompt(">"));
+
+function repeat() {
+    for (let i = 0; i < 10; i++)
```

Comment:
Although the code works as you wrote it, I highly recommend using curly braces {} on `for` loops. It prevents many mistakes in the future and makes the code easier to read.
```js
for (let i = 0; i < 10; i++) {
  // your code
}
```